### ISSUE TITLE
make our dashboard more contributable

### ISSUE THREAD

**Issue by TomasTomecek** [packit/dashboard#209]
yes, contributable is not a word

All our code, deployment recipes, app metadata, documentation is open. The deployment itself (i.e. the cluster) is not for obvious reasons. Metrics are also not open. Hence we meet criteria for dashboard to be a truly [open source service](http://stef.thewalter.net/open-source-services.html).

Let's make dashabord more friendly to receive contributions from the public.

Also, provide info on what exact code (git hash) is running.

TODO:

- [ ] dashboard provides info (commit hash, or git tag) of what exact code it's running
- [ ] every page provides a link to how everyone can contribute
- [ ] (optional) create follow-up cards or provide additional ideas or proposals what we can improve

**Comment by SpyTec** [comment]
> every page provides a link to how everyone can contribute

Like direct to editing or just navigating to the component itself?

**Comment by TomasTomecek** [comment]
This is a follow-up card to my discussion with @ochosi about what he implemented in Image Builder, just to give context.

It's up to us to decide how exactly we want to follow and what the look and implementation be.

> Could display it in the header or bottom of the sidebar.

I'd personally prefer having it in the footer, similar as pagure.io does it.

![image](https://user-images.githubusercontent.com/1662493/216934520-a0d3af4d-1819-4b9a-813c-623da9052a12.png)

Ultimately, this could even evole to a dedicated page where all versions/tags/hashes would be displayed for the whole stack: dashboard, packit-service, packit...

> Like direct to editing or just navigating to the component itself?

I'd just make a single link to a commit in a footer, wouldn't point to components or specific files (in the initial implementation, though this is an interesting suggestion).

@SpyTec what would you suggest?

What are others' thoughts?

**Comment by SpyTec** [comment]
Having a footer that is at the bottom of each page seems easy enough. Can show everything we need there

**Comment by SpyTec** [comment]
> Having a footer that is at the bottom of each page seems easy enough. Can show everything we need there

Actually. If a page get infinite scrolling that auto fetches the next page we run into the issue of never being able to see the footer, relating to the convo in #211

**Comment by TomasTomecek** [comment]
> > Having a footer that is at the bottom of each page seems easy enough. Can show everything we need there
> 
> Actually. If a page get infinite scrolling that auto fetches the next page we run into the issue of never being able to see the footer, relating to the convo in #211

true, that's a good point, we'd need to have it somewhere else. In that case, header makes sense as you show it in the mockup above (and we'd finally start using it, noticed only now that it is empty :D)

**Comment by lbarcziova** [comment]
We have decided to close this and open more helpful follow-ups as needed (like actual refactoring and getting rid of dead code which would help first time contributors).
